src/foremast/utils/awslambda.py

```diff
@@ -39,17 +39,23 @@ def get_lambda_arn(app, account, region):
     session = boto3.Session(profile_name=account, region_name=region)
     lambda_client = session.client('lambda')
 
-    lambda_functions = lambda_client.list_functions()['Functions']
-
-    for lambda_function in lambda_functions:
-        if lambda_function['FunctionName'] == app:
-            lambda_arn = lambda_function['FunctionArn']
-            LOG.debug("Lambda ARN for lambda function %s is %s.", app, lambda_arn)
-            return lambda_arn
-    else:
+    lambda_arn = None
+    paginator = lambda_client.get_paginator('list_functions')
+
+    for lambda_functions in paginator.paginate():
+        if lambda_arn:
+            break
+        for lambda_function in lambda_functions['Functions']:
+            if lambda_function['FunctionName'] == app:
+                lambda_arn = lambda_function['FunctionArn']
+                LOG.debug("Lambda ARN for lambda function %s is %s.", app, lambda_arn)
+                break
```

Comment:
We discussed about how to remove the double `break`. This second level `for` loop would need to be refactored out to a new function, but we're okay with keeping for now.